snuba/admin/clickhouse/common.py

```diff
@@ -24,6 +24,10 @@ class InvalidStorageError(SerializableException):
     pass
 
 
+class ClickHouseTimeoutError(SerializableException):
```

Comment:
styling nit: we generally don't capitalize the H in ClickHouse in variable names e.g.

```suggestion
class ClickhouseTimeoutError(SerializableException):
```